tests/installation/installation_overview.pm

```diff
@@ -62,13 +59,7 @@ sub run {
         assert_screen 'autoyast_removed';
     }
 
-    if (check_screen('manual-intervention', 0)) {
-        $self->deal_with_dependency_issues;
```

Comment:
I think these are still handy for update scenarios. I just wanted broken patterns to be fatal.